.github/workflows/flush.yaml

```diff
@@ -23,4 +23,7 @@ jobs:
       - name: refresh url
         run: |
           qshell account ${{ secrets.QINIU_ACCESS_KEY }} ${{ secrets.QINIU_SECRET_KEY }} tidb
-          qshell cdnrefresh -i cdnfresh.txt
+          result=$(qshell cdnrefresh -i cdnfresh.txt | grep -w "200")
+          if [ -z "$result" ]; then
+            exit 1
+          fi
```

Comment:
A success reponse is `[I]  CDN refresh Code: 200, FlowInfo: success`. So I set it to fail if `200` is not found.